SOURCES/0001-install-into-versioned-dir.patch

```diff
@@ -0,0 +1,16 @@
+diff -uNr a/bnx2fc/driver/Makefile b/bnx2fc/driver/Makefile
+--- a/bnx2fc/driver/Makefile	2018-04-10 17:22:05.504217923 +0000
++++ b/bnx2fc/driver/Makefile	2018-10-22 14:59:27.568195156 +0000
+@@ -251,11 +251,7 @@
+ 
+ bnx2fc_install:
+ 	mkdir -p $(PREFIX)/lib/modules/$(KVER)/kernel/drivers/scsi;
+-ifeq ($(DISTRO), "RHEL")
+-	install -m 444 $(BCM_DRV) $(PREFIX)/lib/modules/$(KVER)/extra;
+-else
+-	install -m 444 $(BCM_DRV) $(PREFIX)/lib/modules/$(KVER)/updates;
+-endif
++	install -m 444 $(BCM_DRV) $(PREFIX)/lib/modules/$(KVER)/updates/xcp-ng/netxtreme2-7.14.46;
```

Comment:
"netxtreme2-7.14.46": there are no variables than you can use to dynamically set the version number?

Next time we update the driver, you can be sure that we'll forget about this path.

If there is no variable that we can use, then you should probably move the file(s) in the %install section of the RPM, where macros are available.